pom.xml

```
@@ -132,6 +132,7 @@
             <groupId>org.jspecify</groupId>
             <artifactId>jspecify</artifactId>
             <version>1.0.0</version>
+            <optional>true</optional>
```

So make it just like `jakarta.validation:jakarta.validation-api`?

```suggestion
            <scope>provided</scope>
            <optional>true</optional>
```

---

Yes, for the record, I also prefer `provided` over `optional`.